src/App.jsx

```diff
@@ -14,33 +15,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [activeSort, setActiveSort] = useState(null);
+  const [isReversed, setIsReversed] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const isOriginalOrder = () => {
+    return JSON.stringify(goods) === JSON.stringify(goodsFromServer);
+  };
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  const sortAlphabetically = () => {
+    const sorted = [...goods].sort((a, b) => a.localeCompare(b));
```

Comment:
The `sortAlphabetically` function creates a sorted copy of the `goods` array. This is correct, but ensure that the `localeCompare` method is sufficient for your sorting needs, especially if you expect to handle international characters.